app/src/main/java/com/infomaniak/swisstransfer/ui/components/Buttons.kt

```diff
@@ -69,7 +69,7 @@ private fun CoreButton(
     enabled: Boolean,
     onClick: () -> Unit,
     imageVector: ImageVector?,
-    titleRes: Int
+    titleRes: Int,
```

Comment:
Maybe add @StringRes as the `SmallButton` function.

You should also maybe keep the same order for parameters on both function (`SmallButton() `and `LargeButton().`